dask/dataframe/io/tests/test_sql.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import print_function, division, absolute_import
+
+import io
+import pytest
+
+from dask.dataframe.io.sql import read_sql_table
+from dask.utils import tmpfile
+pd = pytest.importorskip('pandas')
+dd = pytest.importorskip('dask.dataframe')
+pytest.importorskip('sqlalchemy')
+pytest.importorskip('sqlite3')
+
+
+data = """
+name,number,age,negish
+Alice,0,33,-5
+Bob,1,40,-3
+Chris,2,22,3
+Dora,3,16,5
+Edith,4,53,0
+Francis,5,30,0
+Garreth,6,20,0
+"""
+
+df = pd.read_csv(io.StringIO(data), index_col='number')
+
+
+@pytest.yield_fixture
+def db():
+    with tmpfile() as f:
+        uri = 'sqlite:///%s' % f
+        df.to_sql('test', uri, index=True, if_exists='replace')
+        yield uri
+
+
+def test_simple(db):
+    # single chunk
+    data = read_sql_table('test', db, npartitions=2, index_col='number'
+                          ).compute()
+    assert (data.name == df.name).all()
+    assert data.index.name == 'number'
+    assert (data.columns == df.columns).all()
```

Comment:
Is it reasonable to use `assert_eq` here?  Generally using `assert_eq` checks lots of corner cases that we forget to check otherwise.